source/_integrations/command_line.markdown

```diff
@@ -25,7 +25,7 @@ To use your Command binary sensor in your installation, add the following to you
 # Example configuration.yaml entry
 binary_sensor:
   - platform: command_line
-    command: "cat /proc/sys/net/ipv4/ip_forward"
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
```

Comment:
Inside the command itself, yes, it has no value here in this example here.
Maybe the example is wrong?

Additionally, this now violates our YAML style guide.

Reply:
But the note one line down says:

> It’s highly recommended to enclose the command in single quotes ' as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: ''.

I merely adjusted the example to that note, to resolve the discrepancy. All other examples further down are using single quotes.
So.... is the note wrong? Any suggestions to resolve/way forward?

Reply:
YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

As for this particular example & comment. Quotes used inside the command itself, are different for shell scripts when using single our double quotes (shell expansion).

This is not related to the string quotation in YAML.

Reply:
Thanks!